app/upgrades/v9/upgrades.go

```diff
@@ -67,8 +67,10 @@ func CreateUpgradeHandler(
 				sdk.MsgTypeURL(&authz.MsgRevoke{}),
 				sdk.MsgTypeURL(&gammtypes.MsgJoinPool{}),
 				sdk.MsgTypeURL(&gammtypes.MsgExitPool{}),
-				sdk.MsgTypeURL(&gammtypes.MsgSwapExactAmountIn{}),
-				sdk.MsgTypeURL(&gammtypes.MsgSwapExactAmountOut{}),
+				// N.B.: messsages were moved to another module.
+				// Leaving for historic reasons.
+				// sdk.MsgTypeURL(&gammtypes.MsgSwapExactAmountIn{}),
+				// sdk.MsgTypeURL(&gammtypes.MsgSwapExactAmountOut{}),
```

Comment:
Wait what, I thought were maintaining old message compatability still? Otherwise ICA and wasmbinding users have to migrate?